turbinia/notify.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from turbinia import config
+
+#Sends notfications via email
+def sendmail(message):
+
+    log=logging.getLogger('turbinia')
+    try:
+        EMAIL_NOTIFICATIONS = config.EMAIL_NOTIFCATIONS
+        if EMAIL_NOTIFICATIONS == 'true':
+            EMAIL_HOST_ADDRESS = config.EMAIL_HOST_ADDRESS
+            EMAIL_PORT = int(config.EMAIL_PORT)
+
+            EMAIL_ADDRESS = config.EMAIL_ADDRESS
+            EMAIL_PASSWORD = config.EMAIL_PASSWORD
+
+            RECIEVING_ADDRESS = config.RECIEVING_ADDRESS
+            #Imports SMTP and  MIME(The format used for emails)
+            from email.mime.multipart import MIMEMultipart
+            from email.mime.text import MIMEText
+            import smtplib
+
+            #Puts message in MIME format
+            msg = MIMEMultipart()
+            msg['From'] = EMAIL_ADDRESS
+            msg['To'] = RECIEVING_ADDRESS
+            msg['Subject'] = 'Notifcation from Turbina'
```

Comment:
This would be another thing that would be nice to add as a parameter to the function.